src/background/preferences.ts

```diff
@@ -26,6 +26,7 @@ export class Preferences {
     iconColor: 'default',
     isolation: {
       active: true,
+      autoIsolateDelay: 0,
```

Comment:
I'd like to avoid reusing words the parent preference object already describes (`isolation` vs `Isolate` in this case). Maybe something like `reactivateDelay`?

Reply:
Since the UI uses "Disable" and "Enable" `autoEnableDelay` seems like the right name.